app/controllers/CreateController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+class CreateController extends ApplicationController
+{
+	public function createAction()
+	{
+			if ($this->getRequest()->isPost()) {
+					$title = $_POST['title'] ?? null;
+					$description = $_POST['description'] ?? null;
+					$tags = $_POST['tags'] ?? null;
+					$start_time = $_POST['start_time'] ?? null;
+					$end_time = $_POST['end_time'] ?? null;
+					$created_by = $_POST['created_by'] ?? null;
+					$state = $_POST['state'] ?? null;
+	
+					// Validación de campos
+					if (!$title || !$description || !$tags || !$start_time || !$end_time || !$created_by || !$state) {
+							echo "Error: Faltan datos.";
```

Comment:
I leave here a reminder to manage errors. Here this line prints it out in the frontend, we should probably hide it or consider throwing redirection towards an error page.

Reply:
I've changed it!